app/serializers/recipe/index_serializer.rb

```diff
@@ -8,4 +8,8 @@ def created_at
   def updated_at
     object.created_at.to_formatted_s(:long)
   end
+
+  def user
+    object.user.email
+  end
```

Comment:
I would argue that we don't need users email in response. We don't use it on client side.